src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1274,16 +1274,16 @@
     <Kind Name="AnonymousObjectMemberDeclarator"/>
     <Field Name="NameEquals" Type="NameEqualsSyntax" Optional="true">
       <PropertyComment>
-        <summary>NameEqualsSyntax representing the optional name of the property being initialized.</summary>
+        <summary>NameEqualsSyntax representing the optional name of the member being initialized.</summary>
```

Comment:
Great point. Member it is.

Regards,

-ADG

Sent from my Windows Phone
________________________________
From: Daniel Paoliello<mailto:notifications@github.com>
Sent: ‎5/‎14/‎2015 21:24
To: dotnet/roslyn<mailto:roslyn@noreply.github.com>
Cc: Anthony D. Green<mailto:adgreen@microsoft.com>
Subject: Re: [roslyn] Minor cleanup of C#'s Syntax.xml (#2750)


In src/Compilers/CSharp/Portable/Syntax/Syntax.xml<https://github.com/dotnet/roslyn/pull/2750#discussion_r30381226>:

> @@ -1274,16 +1274,16 @@
>      <Kind Name="AnonymousObjectMemberDeclarator"/>
>      <Field Name="NameEquals" Type="NameEqualsSyntax" Optional="true">
>        <PropertyComment>
> -        <summary>NameEqualsSyntax representing the optional name of the property being initialized.</summary>
> +        <summary>NameEqualsSyntax representing the optional name of the member being initialized.</summary>


Given that the type name was changed from AnonymousObjectPropertyInitializerSyntax to AnonymousObjectMemberDeclaratorSyntax I figured that the comment should have also been updated

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/2750/files#r30381226>.